github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -203,7 +204,7 @@ presubmits:
         - >
             apt update &&
             apt install gettext-base -y &&
-            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz &&
+            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -nv -S &&
```

Comment:
-S
       --server-response
           Print the headers sent by HTTP servers and responses sent by FTP
           servers.


why we need this change ?
(the other is no verbose, why we care for the headers?)